### PR TITLE
Improve xsdir.json creation

### PR DESCRIPTION
**Pull request by jsbull61**
Modified to print the full path of the xsdir file as well as the number of CPUs used to create xsdir.json.
    
Added comments and defined the variable 'N' in GettingStarted.ipynb.

### CODE REVIEW

**Comment by jlconlin** [comment]
Thanks @jsbull61!

While you were working on this, I did something similar, but forgot to push it up. Take a look at it here: https://github.com/NuclearData/DataListing/tree/xsdir and let me know which one you think is better; perhaps a combination of the two.

**Comment by jsbull61** [comment]
I liked your print statement better so I kept that. I also added that statement to ```listing.py``` I also add am error message if the xsdir file wasn't found in both ```listing.py``` and ```GettingStarted.ipynb``` instead of have the code crash.

File: GettingStarted.ipynb [hunk 1]
```diff
@@ -54,12 +62,25 @@
...
+    "datapath = pathlib.Path(os.environ['DATAPATH'], xsdirName)\n",
+    "\n",
+    "if not datapath.is_file():\n",
+    "    print(\"ERROR: xsdir file {}\".format(datapath),\"does not exist\")\n",
+    "else:\n",
+    "    print(\"Reading from xsdir at: {}\".format(datapath))\n",
+    "\n",
+    "# The variable N is how many parallel threads are used\n",
+    "# By default, it uses all but one of the available CPUs\n",
+    "    N = max(1, mp.cpu_count()-1)\n",
```
Comment (jlconlin) [inline GettingStarted.ipynb:80]: Your indentation here is not consistent; it's hard to tell from this interface. Please fix this and then I think It's ready to go. Although, I am still (informally) testing on my end to make sure everything works properly.

Comment (jlconlin) [reply GettingStarted.ipynb:80]: Ah, you're right. Then please also indent the comments.

Comment (jsbull61) [reply GettingStarted.ipynb:80]: Will do.  Do you want the default value of ```xsdirName``` set back to ```xsdir```?

Comment (jsbull61) [reply GettingStarted.ipynb:80]: If ```xsdirName``` is not set here, the cell crashes with a ```name xsdirName is not defined``` error.

Comment (jlconlin) [reply GettingStarted.ipynb:80]: For now, just leave it as `xsdir`. I'll make some changes in a future Pull Request.

**Comment by jsbull61** [comment]
This should be ready to merge into the main repository now.

**Pull request merged**